.github/workflows/codespell.yml

```diff
@@ -0,0 +1,36 @@
+name: Codespell
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  codespell:
+    name: Check Spelling
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Create default packages file
+        run: |
+          cat <<EOF > .mise-python-default-packages
+          codespell==2.4.0
```

Comment:
Why can't we add codespell to mise? And install it directly? because it will bring Python dependencies?

Reply:
AFAIK, there isn't a codespell mise/asdf plugin that we can use.

What this does is have mise install python, then install the codespell package immediately afterwards.